charts/console/templates/deployment.yaml

```
@@ -162,6 +162,97 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
 ---
+{{- if .Values.authentication.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "console.fullname" $ }}-authentication
+  labels:
+    {{- include "console.labels" $ | nindent 4 }}
+    app.kubernetes.io/component: authentication
+spec:
+  replicas: {{ .Values.authentication.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "console.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: authentication
+  template:
+    metadata:
+      {{- with .Values.authentication.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        app.kubernetes.io/component: authentication
+        {{- include "console.selectorLabels" $ | nindent 8 }}
+        {{- with .Values.authentication.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "console.serviceAccountName" $ }}
+      {{- with .Values.authentication.podSecurityContext }}
+      securityContext:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          {{- with .Values.authentication.securityContext }}
+          securityContext:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          image: "{{ .Values.authentication.image.repository }}:{{ .Values.authentication.image.tag }}"
```

To enable the tag to default to the chart appVersion
```suggestion
          image: "{{ .Values.authentication.image.repository }}:{{ .Values.authentication.image.tag | default .Chart.AppVersion }}"
```